src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ total }) => <div className="display">{total || '0'}</div>;
+
+Display.propTypes = {
+  total: PropTypes.string,
+};
+
+Display.defaultProps = {
+  Display: '.',
```

Comment:
@udberg, Perhaps this is a mistake.

I think it should be `total: '0'`

we are trying to write defaultProps for our props called `total`.
So I see nowhere you have `Display`.


@udberg kindly reply here, if you're online, so that you won't need to submit another CRR.